images/win/scripts/Tests/MSYS2.Tests.ps1

```diff
@@ -0,0 +1,21 @@
+BeforeAll {
+    $msys2BinDir = "C:\msys64\usr\bin"
+    $msys2mingwDir = "C:\msys64\mingw64\bin"
+    }
+Describe "MSYS2" {
+    It "<ToolsInstalled>" -TestCases @(
+        @{ ToolsInstalled = "bash.exe" }
```

Comment:
What `ToolsInstalled` means here? May be `ToolName` will be clearer? Looks like in other files you use `ToolName`

Reply:
Agree, fixed.